tests/generic_config_updater/test_multiasic_scenarios.py

```diff
@@ -1,217 +0,0 @@
-    # Updated regex pattern to match both (U) and (S) status
-    pattern = re.compile(r"^\s*(\d+)\s+(PortChannel\d+)\s+LACP\(\w+\)\(Up\)\s+(Ethernet\d+)\([US]\)", re.MULTILINE)
-    match = pattern.search(output)
-    if match:
-        return match.group(2), match.group(3)
-    else:
-        return None, None
-
-
-@pytest.fixture(autouse=True)
-def setup_env(duthosts, rand_one_dut_hostname):
-    """
-    Setup/teardown fixture for each multi asic test.
-    rollback to check if it goes back to starting config
-    Args:
-        duthosts: list of DUTs.
-        rand_selected_dut: The fixture returns a randomly selected DuT.
-    """
-    duthost = duthosts[rand_one_dut_hostname]
-
-    create_checkpoint(duthost)
-
-    yield
-
-    try:
-        logger.info("Rolled back to original checkpoint")
-        rollback_or_reload(duthost)
-    finally:
-        delete_checkpoint(duthost)
-
-
-def test_check_empty_apply_patch(duthost):
-    json_patch = []
-    tmpfile = generate_tmpfile(duthost)
-
-    try:
-        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
-    finally:
-        delete_tmpfile(duthost, tmpfile)
-
-    if output['rc'] or "Patch applied successfully" not in output['stdout']:
-        logger.info("Patching process broken, the error output is {}").format(output['stdout'])
-        pytest_assert(False, "Patching process broken, the error output is {}").format(output['stdout'])
-
-
-def test_check_idf_isolation_apply_patch(duthost):
-    json_patch = IDF_ISOLATION
-    tmpfile = generate_tmpfile(duthost)
-
-    try:
-        print("The current running config is:")
-        print(duthost.shell("show run all", module_ignore_errors=False)['stdout'])
-        logger.info("The current running config is:")
-        logger.info(duthost.shell("show run all", module_ignore_errors=False)['stdout'])
-
-        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
-
-        if output['rc'] or "Patch applied successfully" not in output['stdout']:
-            logger.info("Patching process broken, the error output is {}".format(output['stdout']))
-            pytest_assert(False, "Patching process broken, the error output is {}").format(output['stdout'])
-
-        cmds = 'sonic-db-cli -n asic0 CONFIG_DB hget "BGP_DEVICE_GLOBAL|STATE" idf_isolation_state'
-        expected_value = "isolated_no_export"
-        redis_value = duthost.shell(cmds, module_ignore_errors=False)['stdout']
-        pytest_assert(redis_value == expected_value, "Config IDF ISOLATION failed")
-
-        cmds = 'sonic-db-cli -n asic1 CONFIG_DB hget "BGP_DEVICE_GLOBAL|STATE" "idf_isolation_state"'
-        expected_value = "isolated_withdraw_all"
-        redis_value = duthost.shell(cmds, module_ignore_errors=False)['stdout']
-        pytest_assert(redis_value == expected_value, "Config IDF ISOLATION failed")
-    finally:
-        delete_tmpfile(duthost, tmpfile)
-
-
-def test_check_idf_unisolation_apply_patch(duthost):
```

Comment:
why this testcase is removed in new iteration?

Reply:
Per comments, I split the original test file into two separate scenarios. This test is in [test_multiasic_idf.py](https://github.com/sonic-net/sonic-mgmt/pull/13210/files#diff-0ee009a322253bd061db5b786fdbbd1a93aa3f89d947d02a950160fc0eb855cb)